### ISSUE TITLE
Is PageTypeRoute attribute deprecated?

### ISSUE THREAD

**Issue by jasonterando** [PiranhaCMS/piranha.core#1874]
Hi, I found [this Piranha tutorial page]https://piranhacms.org/blog/techniques-for-securing-pages) on securing pages using Piranha CMS.  It refers to an attribute called PageTypeRoute which looks like it got added [in this commit back in 2017](https://github.com/PiranhaCMS/piranha.core/commit/1f583c608df9136f7f580b7dc844739143dfa780), but is no longer in the [master](https://github.com/PiranhaCMS/piranha.core/tree/master/core/Piranha.AttributeBuilder) branch.

Did this attribute get moved or is there an alternate technique I should be using to set up per-page authorization?

Thanks!

**Comment by tidyui** [comment]
Yes it’s depcrecated! Now it’s called ContentTypeRouteAttribute and is used for both pages and posts. I’ll move this issue to the documentation repo and rename it for clarity.
